lib/caesar.rb

```diff
@@ -1,6 +1,24 @@
+require "pry"
 class Caesar
-  attr_reader :key
+
 
   def initialize
+
+  end
+
+
+  def caesar_cipher(string, shift)
+    array = Array.new
+    array << string.chars
+
+    array.map do |strings|
+      strings.ord
+    end
+
+    while array != nil
+      array.rotate!(shift)
+    end
+
+    array.chr
   end
```

Comment:
I would suggest breaking this into multiple methods. For instance, if there is one mistake in your caesar_cipher method, the whole thing won't work. Try to break the problem into smaller pieces.